### ISSUE TITLE
collection_tesim vs. archival_collection_tesim

### ISSUE THREAD

**Issue by jgreidy** [cul-it/singlecore-library-cornell-edu#1023]
The MAP spreadsheet has a field called 'Archival Collection' with a 'New _SOLR field name' of 'archival_collection_tesim'. 
There are about 500 places in the singlecore code that rely on the old name 'collection_tesim'. They could be globally replaced in the code, but many of these references are in URLs. This would affect many canned searches and links that are out in the world.

Can we change the solr name back to collection_tesim? We can leave the display name as 'Archival Collection'.

**Comment by jgreidy** [comment]
Maybe a solution would be to copyfield the archival_collection_tesim field into collection_tesim in solr so we can continue using the old name in the code.

**Comment by mhk33** [comment]
Archival Collection is different that Collection. We need both.

**Comment by mhk33** [comment]
It is weird that I don't see it displaying in the portal right now; it used to. Here's an example record where it should be displaying:
https://digital.library.cornell.edu/catalog/ss:3317772
Project 167

I wonder if something happened in the indexing or if something changed in Forum. Archival Collection is an important field for Rare collections.

**Comment by cm647** [comment]
ARCHIVAL COLLECTION!
Here are all the collections (in shorthand) that use a controlled list with the field “Archival Collection” in JStor Forum:
Prison Reform
Bambaataa
Alison Mason Kingsbury
AD White
Adler
Claire Holt
Ahearn
Racker Paintings
Pannicoli
Fuertes
Hedda Morrison
Hip Hop Flyers
Glaciers
Icelandic-Cambridge
Icelandic-Stereoscopes
Bastides
Reps Slides
Lindsay Cooper
Lowenthiel
Nuclear Fallout
PJ Mode
Impersonator Postcards
Punk Flyres
RMC
Rudin
Sterrett

Here is a collection that usese the same controlled list, but the field label is “Archival Collection Title”
Conzo

Here are a few that use “Archival Collection” but have a string value:
Gems
WCMC History of Psychiatry
KARMA

If there’s other useful stuff to provide, let me know
Chloe

Chloe A. McLaren
Metadata Projects Librarian
Library Technical Services
Cornell University Library

**Comment by jgreidy** [comment]
In the MAP spreadsheet there is 'Artstor Collection Name' with the solr lookup field 'collection_tesim', but JSTOR does not return a value for this field in the metadata for any collection.

**Comment by mhk33** [comment]
Yeah, I don't think we've been able to get the actual collection name from Forum in the past. We've been setting it manually in the ini.

**Comment by jgreidy** [comment]
In ss2solr.hiphopflyers.ini there is this line:
set_solr_field[collection_tesim] = "Hip Hop Party and Event Flyers"

These were solr fields we added manually per collection in the legacy system - they are not coming from JStor Forum. It seems like this sort of information should come from JStor, so metadata people can control it. It's 'Collection Level Metadata'.

Other examples: 
set_solr_field[collection_website_ss] = "http://rmc.library.cornell.edu/hiphop/flyers.html"
set_solr_field[ssc_site_tesim] = "http://hdl.handle.net/1813.001/8002837a"
set_solr_field[format_tesim] = "Image"
set_solr_field[bibid_ssi] = "8002837"

**Comment by mhk33** [comment]
I thought we had this discussion already and were going to store this info in a spreadsheet and pull that into SOLR. And there wasn't a way to store this collection-level info in Forum.

**Comment by jgreidy** [comment]
I found the spot in the MAP code where this is handled - it was already taken care of. I added handlers for collection_tesim and archival_collection_tesim to catalog_controller.rb

For 3317772, I see Archival Collection coming back from Forum:
array:75 [
  "archival_collection_tesim" => array:5 [
    "forum" => "fd_4556_mfcl_lookup"
    "field" => "Archival Collection"
    "solr" => "archival_collection_tesim"
    "value" => "Breakbeat Lenny Archive"
    "copyfields" => array:1 [
      0 => "* all_text_timv"
    ]
  ]
